modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -91,6 +91,20 @@ definition {
 				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
 		}
 
+		task ("And When guest user toggles show underline to OFF") {
+			Navigator.openURL();
+
+			AccessibilityMenu.openAccessibilityMenu();
+
+			Uncheck.uncheckToggleSwitch(
+				key_toggleSwitchLabel = "Underlined Links",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+				
+			AssertNotChecked.assertNotCheckedNotVisible(
+				key_toggleSwitchLabel = "Underlined Links",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+				
+		}
 		task ("And When guest user toggles show underline to ON") {
 			Navigator.openURL();
```

Comment:
Not sure if `Navigator.openURL();` is actually needed at this point, what do you think?

Reply:
I would keep it.

In theory, it shouldn't be needed because toggled changes should be saved and applied immediately but...I don't want to count on it for this test...